tests/integration/test_remote_transfer.py

```
@@ -141,16 +141,13 @@ async def test_push_transfer_success(
         assert job.status == TransferStatus.COMPLETED
 
 
-@respx.mock
-async def test_pull_transfer_success(
+def test_pull_transfer_legacy_endpoint_removed(
     db_session: Session,
```

Ruff (0.15.12)

[warning] Unused function argument: `db_session`

(ARG001)